src/view/TrackView.py

```diff
@@ -0,0 +1,140 @@
+from __future__ import annotations
+
+from typing import List, Optional, Type, Dict, Any
+
+import pygame
+from planar import Vec2
+from pygame.event import EventType
+from pygame.rect import Rect
+from pygame.surface import Surface
+
+from model.Simulation import Simulation
+from model.geom.track import Track
+from view import Colors
+from view.Action import Action, ActionType
+from view.View import View
+
+
+class TrackView(View):
+    simulation: Simulation
+    simulation_class: Type[Simulation]
+    dataset: Dict[str, Any]
+    board: Surface
+    coord_start: Vec2
+    scale = 1.0
+    poi = Vec2(0, 0)
```

Comment:
What does `poi` mean?

Reply:
Point of interest, central point to focus screen on. In later revision will be set to the winning car position. Name may be changed to make it self describing, Real acronym should be capitalized but that would conflict with python naming conventions